src/CreatureList.js

```diff
@@ -0,0 +1,18 @@
+import React, { Component } from 'react';
+import CreatureItem from './CreatureItem';
+import './CreatureList.css';
+import Creatures from './Creatures';
+
+class CreatureList extends Component {
+
+  render() {
+    return (
+      <ul className="CreatureList">
+        {Creatures.map(creature => (
+          <CreatureItem key={creature.title} creature={creature} />
+        ))}
+      </ul>
```

Comment:
Rather than importing your creature data, you want to pass down your creatures prop from App.js. On line 8.5, you would say something like `const creatures = this.props.creatures` and from there, map through `creatures`.